demo/animations.html

```diff
@@ -1,153 +1,122 @@
-			</ul>
-		</p>
-	</div>
-
-	<article id="entering">
-		<h2 class="lui divider below">Entering animations</h2>
-
-		<div class="lui dashed divider">
-			<h4 class="lui underline">Usage:</h4>
+	<div id="main-content" class="lui container">
+		<h1 class="lui column desktop-8">Lucca UI</h1>
+		<div class="lui column desktop-4">
+			<a class="lui large button pull-right" href="https://github.com/LuccaSA/lucca-ui">Code on github</a>
 		</div>
-		<p>
-			<pre ng-non-bindable>&lt;ANY class="lui animated up fade in"&gt;&lt;/ANY&gt;
-&lt;ANY class="lui animated _direction_ _animation_"&gt;&lt;/ANY&gt;</pre>
-		</p>
-		<div class="lui dashed divider">
-			<h4 class="lui underline">Demo:</h4>
-		</div>
-		<div ng-controller="enterCtrl" class="lui block round">
+		<p class="lui sub heading">DOCUMENTATION - Getting up and running with the framework</p>
+
+		<div class="lui callout primary">
 			<p>
-				<div class="lui buttons">
-					<button class="lui button" ng-model="animation" uib-btn-radio="'fade in'">Fade in</button>
-					<button class="lui button" ng-model="animation" uib-btn-radio="'slide in'">Slide in</button>
-					<button class="lui button" ng-model="animation" uib-btn-radio="'scale in'">Scale in</button>
-				</div>
-				<div class="lui buttons">
-					<button class="lui button" ng-model="direction" uib-btn-radio="'up'">Up</button>
-					<button class="lui button" ng-model="direction" uib-btn-radio="'down'">Down</button>
-					<button class="lui button" ng-model="direction" uib-btn-radio="'left'">Left</button>
-					<button class="lui button" ng-model="direction" uib-btn-radio="'right'">Right</button>
-				</div>
-				<div class="lui button" ng-click="add()">Add</div> <code>lui animated {{direction}} {{animation}}</code>
+				<b>Important:</b>
+				The js part of lucca-ui is an angular framework so is obviously dependent on <code>angular</code>.
+				It also uses <code>momentjs</code> for its manipulation of dates cuz <a href="http://momentjs.com/">momentjs</a> is super awesome
 			</p>
 			<p>
-				<span ng-repeat="item in array track by $index" class="lui label animated" ng-class="direction + ' ' + animation">{{item}}</span>
+				The exact list of dependencies which can be found in the bower.json file is:
+				<ul class="lui unstyled">
+					<li><a href="https://github.com/LuccaSA/lucca-icons">lucca-icons</a> <span class="lui label">1.1</span> - this dependency is not used here</li>
+					<li><a href="https://angularjs.org/">angular</a> <span class="lui label">1.4</span> - this dependency is not used here</li>
+					<li><a href="https://angular-ui.github.io/bootstrap/">angular ui bootstrap</a> <span class="lui label">0.13</span> - this dependency is not used here</li>
+					<li><a href="http://momentjs.com/">momentjs</a> <span class="lui label">2.10</span> - this dependency is not used here</li>
+				</ul>
 			</p>
 		</div>
-	</article>
```

Comment:
I also want to add the callout in the main page and not copy paste it in each page but i'm not quite sure how to do it well now because the list of dependencies is not the same for each page

also some messages only apply to a particular page - `#/less` does not care about no angularjs or momentjs